frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/compare-patients.ts

```diff
@@ -0,0 +1,58 @@
+import type {
+    ExerciseConfiguration,
+    PatientStatus,
+} from 'digital-fuesim-manv-shared';
+import { Patient } from 'digital-fuesim-manv-shared';
+
+const patientCategoryOrderDictionary: {
+    [Key in PatientStatus]: number;
+} = {
+    black: 5,
+    blue: 4,
+    green: 3,
+    red: 1,
+    white: 0,
+    yellow: 2,
```

Comment:
Could you please order these by value? :)

Reply:
Done in https://github.com/hpi-sam/digital-fuesim-manv/pull/739/commits/74c239a34b455480ce4561bf14561e985ad897be